hooks/useUserStakingData.ts

```diff
@@ -108,18 +123,25 @@ export const useUserStakingData = (walletAddress: string | null) => {
 		};
 	}, [currentFeePeriod]);
 
-	const lastClaimDate = new Date(
-		feeClaims.isSuccess && feeClaims.data.length ? feeClaims.data[0].timestamp.toNumber() : 0
-	);
+	const lastClaimDate = new Date(feeClaimsData?.length ? feeClaimsData[0].timestamp.toNumber() : 0);
 
 	const hasClaimed = lastClaimDate > currentFeePeriodStarts && lastClaimDate < nextFeePeriodStarts;
 
-	function refetch() {
-		feeClaims.refetch();
-		exchangeRatesQuery.refetch();
-		lockedSnxQuery.refetch();
-		debtData.refetch();
-	}
+	const refetch = useCallback(() => {
+		feeClaimsRefetch();
+		exchangeRatesRefetch();
+		lockedSnxRefetch();
+		debtRefetch();
+		availableRewardsRefetch();
```

Comment:
This was the real culprit @bachstatter @noisekit

Reply:
NICE!